lib/facebook_app_events.dart

```diff
@@ -17,7 +17,7 @@ class FacebookAppEvents {
   static const eventNameViewedContent = 'fb_mobile_content_view';
   static const eventNameRated = 'fb_mobile_rate';
   static const eventNameInitiatedCheckout = 'fb_mobile_initiated_checkout';
-
+  static const evanetNameAddToCart = 'fb_mobile_add_to_cart';
```

Comment:
Refactor rename spelling: `eventNameAddToCart`

Reply:
I have made the required changes.